package.json

```diff
@@ -82,19 +82,19 @@
 		"build-docker": "node bin/build-docker.js",
 		"build-static": "npx ncp static public && npx ncp client/lib/service-worker public --filter=\"service-worker(/service-worker\\.js)?$\"",
 		"prebuild-server": "mkdirp build",
-		"build-server": "BROWSERSLIST_ENV=server webpack --config client/webpack.config.node.js --stats-preset errors-only",
+		"build-server": "BROWSERSLIST_ENV=server webpack --config client/webpack.config.node.js --stats-preset errors-only $WEBPACK_OPTIONS",
 		"build-server:copy-modules": "node bin/copy-production-modules.js",
 		"postbuild-server": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || yarn run build-server:copy-modules",
 		"build-server-stats": "NODE_ENV=production CALYPSO_ENV=production EMIT_STATS=true yarn run build-server",
```

Comment:
Shouldn't we also pass the `WEBPACK_OPTIONS='--progress=profile'` here as well?

Reply:
It is already set as an env var in the Dockerfile (https://github.com/Automattic/wp-calypso/pull/51269/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R23), so building both the server and the client in CI should show the profile.